dist/main.js

```diff
@@ -0,0 +1 @@
="to-do-check ${e.completed}" type="checkbox">\n      <p class="to-do-des" contentEditable="false">${e.description}</p>\n    </div>\n    <i class="fa-solid fa-ellipsis-vertical" id="${e.index}"></i>\n    <i class="fa-solid fa-trash-can"></i>\n    `,v.appendChild(t)}static removeItem(e){e.classList.contains("remove")&&e.parentElement.remove()}static clearInput(){f.value=""}}class b{static updateIndex(){const e=h.getItemStorage();let t=0;for(const n of e)t=e.indexOf(n),n.index=t+1;localStorage.setItem("items",JSON.stringify(e))}static updateDescription(e,t){const n=h.getItemStorage();for(const r of n)n.indexOf(r)==t&&(r.description=e);localStorage.setItem("items",JSON.stringify(n))}}const E=class{static updateStorageCheck(e,t){const n=h.getItemStorage();for(const r of n){const a=n.indexOf(r);e.classList.contains("true")&&t==a+1?r.completed=!0:e.classList.contains("false")&&t==a+1&&(r.completed=!1)}localStorage.setItem("items",JSON.stringify(n))}static clearAllCompleted(){const e=h.getItemStorage();for(let t=0;t<=e.length;t++)e.forEach(((t,n)=>{!0===t.completed&&e.splice(n,1)}));localStorage.setItem("items",JSON.stringify(e))}};class S{constructor(e,t,n){this.completed=e,this.description=t,this.index=n}}const x=document.getElementById("to-do-input"),I=document.getElementById("to-do-list"),L=document.getElementById("clear-completed");y.displayItems(),x.addEventListener("keypress",(e=>{const t=x.value;if("Enter"===e.key&&""!==t){const e=new S(!1,t);y.addItem(e),h.addItemStorage(e),y.clearInput(),b.updateIndex(),window.location.reload()}})),I.addEventListener("click",(e=>{if(!0===e.target.classList.contains("remove")&&(y.removeItem(e.target),h.removeItemStorage(e.target.previousElementSibling.previousElementSibling.lastElementChild.innerHTML),b.updateIndex()),!0===e.target.classList.contains("list-items-c")){e.target.classList.toggle("new-background"),e.target.firstElementChild.nextElementSibling.classList.toggle("item-removed"),e.target.firstElementChild.nextElementSibling.nextElementSibling.classList.toggle("remove"),e.target.firstElementChild.lastElementChild.contentEditable="true";const t=e.target.firstElementChild.nextElementSibling.getAttribute("id");e.target.firstElementChild.lastElementChild.addEventListener("keypress",(e=>{if("Enter"===e.key){e.target.contentEditable="false";const n=e.target.innerHTML;e.target.parentElement.parentElement.classList.toggle("new-background"),e.target.parentElement.nextElementSibling.classList.toggle("item-removed"),e.target.parentElement.nextElementSibling.nextElementSibling.classList.toggle("remove"),b.updateDescription(n,t)}}))}if(!0===e.target.classList.contains("to-do-check")){e.target.parentElement.classList.toggle("line-through"),e.target.classList.toggle("false"),e.target.classList.toggle("true");const t=e.target.parentElement.nextElementSibling.getAttribute("id");E.updateStorageCheck(e.target,t)}})),L.addEventListener("click",(()=>{E.clearAllCompleted(),b.updateIndex(),window.location.reload()}))})()})();
```

Comment:
- [ ] Kindly put the document in order, line by line. It is easier to review and use the inline comments in the corresponding place. 📑 👀